src/applications/common/schemaform/save-in-progress/SaveInProgressErrorPage.jsx

```diff
@@ -6,6 +6,7 @@ import { withRouter } from 'react-router';
 import {
   LOAD_STATUSES,
   PREFILL_STATUSES,
+  PRESTART_STATUSES,
```

Comment:
It seems...not right for this to be in the `SaveInProgressErrorPage`. Like, we've got a thing that we want to do before we start the form. It has nothing to do with SiP. But oh noes! The thing went wrong and we need to show an error page. The SiP error page is convenient, but doesn't seem like it should have its purpose expanded to include errors that happen outside of SiP.

Reply:
That's an interesting point @cvalarida, @kreek has explained that whenever the form is started or resumed (both are handled by `fetchInProgressForm`, within which @jbalboni has advised handling these types of gating checks), we will want to check the ITF, and there are at least three types of scenarios to consider:
1. when the user is on the introduction page or on any of the static pages with the SIP Application Status card and tries to either start, or resume their application
2. Once a form is "saved" if the user is on the form saved page (there is a SIP Application Status card there which enables that as well)
3. if the user tries to resume the form from the user profile page (this page has it's own version of the Application Status card, which leverages the RoutedSavableApp to handle all of the routing/loading/error handling logic)
The idea was to have a consistent error experience for all of these entry points, and to make use of the existing error flow that is already in place for forms.

Reply:
Is it possible to limit the ITF to just being sent at the start of the form, and assumed to be in place after that? That would make this much simpler, otherwise, like Claire says, you have to somehow integrate this check with all the entry points to the form.

Reply:
Thanks @jbalboni, @kreek indicated that we must check this every time they enter/re-enter the form.